File: java/src/main/java/deltix/luminary/implementation/EnumerationMemberNode.java
package deltix.luminary.implementation;

import deltix.luminary.Literal;

import java.util.List;

public class EnumerationMemberNode extends ItemNode {
    private Literal value;

    public EnumerationMemberNode(String name, List<String> comments) {
        super(name, comments);
    }

    public Literal getValue() {
        return value;
    }

    void setValue(Literal value) {
        this.value = value;
    }
}

File: csharp/main/Implementation/EnumerationMemberNode.cs
using System;
using System.Collections.Generic;
using JetBrains.Annotations;

namespace Deltix.Luminary.Implementation
{
	internal class EnumerationMemberNode : ItemNode
	{
		public EnumerationMemberNode([NotNull] String name, [CanBeNull] List<String> comments)
			: base(name, comments)
		{}

		public Literal Value { get; internal set; }
	}
}
